vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2618,6 +2619,25 @@ uint64_t getMediaDuration(const std::map<std::string, std::string>& data,
   return duration;
 }
 
+// static
+bool isValidLeadingUrlSubString(
```

Comment:
nit: we should be using chromium style for all new code - `isValidLeadingUrlSubString`

Reply:
Fixed